docs/pages/machine-id/reference/cli.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Machine ID CLI Reference
+description: CLI reference for Teleport Machine ID.
+---
+
+## `tbot start`
+
+Starts the Machine ID client `tbot`, fetching and writing certificates to disk
+at a set interval.
+
+<Details scope={["cloud"]} scopeOnly={true}>
+```code
+$ tbot start \
+   --data-dir=/var/lib/teleport/bot \
+   --destination-dir=/opt/machine-id \
+   --token=00000000000000000000000000000000 \
+   --join-method=token \
+   --ca-pin=sha256:1111111111111111111111111111111111111111111111111111111111111111 \
+   --auth-server=example.teleport.sh:443
+```
+</Details>
+<Details scope={["oss","enterprise"]} scopeOnly={true}>
+```code
+$ tbot start \
+   --data-dir=/var/lib/teleport/bot \
+   --destination-dir=/opt/machine-id \
+   --token=00000000000000000000000000000000 \
+   --join-method=token \
+   --ca-pin=sha256:1111111111111111111111111111111111111111111111111111111111111111 \
+   --auth-server=auth.example.com:3025
+```
+</Details>
+
+| Flag                 | Description                                                                                    |
+|----------------------|------------------------------------------------------------------------------------------------|
+| `-d/--debug`         | Enable verbose logging to stderr.                                                              |
+| `-c/--config`        | Path to a configuration file. Defaults to `/etc/tbot.yaml` if unspecified.                     |
+| `-a/--auth-server`   | Address of the Teleport Auth Server (On-Prem installs) or Proxy Server (Cloud installs).       |
+| `--token`            | A bot join token, if attempting to onboard a new bot; used on first connect.                   |
+| `--ca-pin`           | CA pin to validate the Teleport Auth Server; used on first connect.                            |
+| `--data-dir`         | Directory to store internal bot data. Access to this directory should be limited.              | 
+| `--destination-dir`  | Directory to write short-lived machine certificates.                                           |
+| `--certificate-ttl`  | TTL of short-lived machine certificates.                                                       |
+| `--renewal-interval` | Interval at which short-lived certificates are renewed; must be less than the certificate TTL. |
+| `--join-method`      | Method to use to join the cluster, can be `token` or `iam`.                                    |
+| `--one-shot`         | If set, quit after the first renewal.                                                          |
+
+## `tbot init`
+
+If you want to write certificates to disk as a different user than the Machine
```

Comment:
It might be good to mention an example in this paragraph of when you would use this so a user scrolling down this page can have a better idea of whether this command is relevant for their use case.